src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator/github-connector.css

```diff
@@ -0,0 +1,79 @@
+.create-github-api-key-container {
```

Comment:
Agree, however strongly opposed to using ids for this. Much of this really should just go away with upcoming UI work.